sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilderSamples.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Samples
+{
+    /// <summary>
+    /// Samples for QueryBuilder.
+    /// </summary>
+    public static class QueryBuilderSamples
+    {
+        /// <summary>
+        /// Main method.
+        /// </summary>
+        public static void main()
+        {
+            #region Snippet:DigitalTwinsQueryBuilder
+
+            // SELECT * FROM DIGITALTWINS
+            AdtQueryBuilder sample1 = new AdtQueryBuilder().Select("*").From(AdtCollection.DigitalTwins).Build();
+
+            // SELECT TOP(3) FROM DIGITALTWINS
+            AdtQueryBuilder sample2 = new AdtQueryBuilder()
+                .SelectTop(3)
+                .From(AdtCollection.DigitalTwins)
+                .Build();
+
+            // SELECT COUNT() FROM RELATIONSHIPS
+            AdtQueryBuilder sample3 = new AdtQueryBuilder()
+                .SelectCount()
+                .From(AdtCollection.Relationships)
+                .Build();
+
+            // SELECT * FROM DIGITALTWINS WHERE IS_OF_MODEL("dtmi:example:room;1")
+            AdtQueryBuilder sample4 = new AdtQueryBuilder()
+                .Select("*")
+                .From(AdtCollection.DigitalTwins)
+                .WhereIsOfModel("dtmi:example:room;1")
```

Comment:
What happened to the idea of `Where()` being an initial clause?

Something like:

```csharp
 AdtQueryBuilder sample4 = new AdtQueryBuilder()
                .Select("*")
                .From(AdtCollection.DigitalTwins)
                .Where()
                .IsOfModel("dtmi:example:room;1")
                .Build();
```

Is that idea still in the works? Did it not work out? Did we reject it?

Reply:
It's still in the works, it will be addressed in the next card on the storyboard that deals with the `Where()` clause. Still not sure if it will be implemented like that but that's when the decision will be made.